src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/factory/StandardPlayerRequestResolverFactory.java

```diff
@@ -51,11 +51,11 @@ public StandardPlayerRequestResolver getNewInstance(
                                                  @NotNull final Object... context)
       throws IllegalArgumentException
     {
-        ILocation location;
+        final ILocation location;
         try {
             location =
             factoryController.getNewInstance(TypeConstants.ILOCATION, iRequestManager.getColony().getCenter(), iRequestManager.getColony().getWorld().provider.getDimension());
-        } catch (Exception ex)
+        } catch (final Exception ex)
         {
             throw ex;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add logic to this catch clause or eliminate it and rethrow the exception automatically. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2737)